pom.xml

```diff
@@ -426,8 +426,8 @@
         <scope>test</scope>
         <exclusions>
           <exclusion>
-            <groupId>javax.servlet</groupId>
             <artifactId>javax.servlet-api</artifactId>
+            <groupId>javax.servlet</groupId>
```

Comment:
I'm surprised the plugin reordered these. Usually, the standard is to have groupId first.

Reply:
I had assumed the plugin knew better that I did, but I agree, this seems strange.